### PR TITLE
Fix term splitting for featured content

### PR DESCRIPTION
**Pull request by samhotchkiss**

### CODE REVIEW

File: modules/theme-tools/featured-content.php [hunk 1]
```diff
@@ -578,4 +578,16 @@ public static function switch_theme() {
 
 Featured_Content::setup();
 
+function jetpack_update_featured_content_for_split_terms( $old_term_id, $new_term_id, $term_taxonomy_id, $taxonomy ) {
+    $fc_settings = get_option( 'featured-content', array() );
```
Comment (enejb) [inline modules/theme-tools/featured-content.php]: I would spell out fc.

File: modules/theme-tools/featured-content.php [hunk 2]
```diff
@@ -578,4 +578,16 @@ public static function switch_theme() {
 
 Featured_Content::setup();
 
+function jetpack_update_featured_content_for_split_terms( $old_term_id, $new_term_id, $term_taxonomy_id, $taxonomy ) {
```
Comment (enejb) [inline modules/theme-tools/featured-content.php]: could we move this function inside the Feature Content Class?

**Comment by enejb** [comment]
Looking though the code I wonder if it is even needed to store the term_id at all. 

I think a better fix would be to remove the need to have the term_id. I am not sure what advantage there is to even storing the term_id in the settings as opposed to just the tag name. @georgestephanis

**Comment by dereksmart** [comment]
@enejb, I made your inline suggestions, and I think we can take a look at refactoring what we store in the future.  For now this should be included in preparation for WP 4.3

**Pull request merged**